src/main/docs/guide/openApiViews/mappingPath.adoc

```diff
@@ -5,13 +5,13 @@ Thus, by default, the views expect to find the `yaml` under `/swagger`.
 If you change this mapping to something else:
 
 .Exposing Swagger YAML
-[configuration]
+[source,yaml]
```

Comment:
we want to keep using the `configuration` macro

Reply:
fixed